### PR TITLE
Add dependencies feedstock

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dhirschfeld** [comment]
ping @wolfv!

**Comment by wolfv** [comment]
nice!

File: recipes/dependencies/bld.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+setlocal enableextensions
+setlocal enabledelayedexpansion
+
+
+xcopy %SRC_DIR%\dependencies %LIBRARY_PREFIX%\dependencies /E /I /Y
```
Comment (isuruf) [inline recipes/dependencies/bld.bat:5]: I think we should copy all of the contents into `%LIBRARY_BIN%`

**Comment by isuruf** [comment]
This seems to be vendoring pakages such as vs2015_runtime and `mono.cecil`

**Comment by dhirschfeld** [comment]
> This seems to be vendoring packages such as vs2015_runtime and mono.cecil

Yes, at the moment this is just packaged as a standalone application. In an ideal world, I suppose it would be built from source with the [`mono`](https://github.com/conda-forge/mono-feedstock) compiler.

I think that *should* be possible but since it will only ever be used as a standalone application I'm not sure there's much point in doing so?

**Comment by isuruf** [comment]
The point is that we don't know what the license of the package is going to be. We'll have to track down the license of each dependency. For eg: what's the license of `mono.cecil`? What are the licenses of the dependencies of `mono.cecil`? Are they statically linked in?

**Comment by dhirschfeld** [comment]
Fair enough. The third party licenses don't seem to be well documented in the upstream repo:
https://github.com/lucasg/Dependencies/tree/master/third_party

Unfortunately it looks like a bit much effort to reverse engineer that so I'll go ahead and close. It'll keep in GitHub history until such time as someone cares to revisit. If they move to a .NET5 build system I might do so myself...

**Comment by isuruf** [comment]
> Unfortunately it looks like a bit much effort to reverse engineer that so I'll go ahead and close.

Then we can ask upstream about the licenses of the third party dependencies since they know more about dependencies (:wink:).

**Pull request closed without merging**